### PR TITLE
Draft shortcuts, long pain

### PR DESCRIPTION
**Pull request by iamvery**

### CODE REVIEW

File: _drafts/shortcut-long-pain.markdown [hunk 1]
```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Shortcut, Long Pain
+---
+
+Programmers are often tempted to take the shortest route possible to deliver a
+feature. While there is a lot to be said about pragmatism, you should be
+considerate of consequences to make the best decision possible.
```
Comment (tdouce) [inline _drafts/shortcut-long-pain.markdown]: I think the wording of this sentence can be improved upon. It's not entirely clear to me. Perhaps something along the lines of:

```
While there is a lot to be said about pragmatism, every decision you make has consequences. So, make the best decision possible
```

Comment (iamvery) [inline _drafts/shortcut-long-pain.markdown]: Great suggestion. I agree, it will make this better.

File: _drafts/shortcut-long-pain.markdown [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+![Convenience of code hinders clarity](/img/blog/2015/02/convenience-hinders-clarity.jpg)
+
+## Shortcut
+
+You are hard at work on a web application, and you're given a feature to create
+a view for pet management.
```
Comment (tdouce) [inline _drafts/shortcut-long-pain.markdown]: Similarly, I think this sentence can use some clarification. Perhaps: 

```
You are given a feature to complete in a web application, create a view for pet management, and you are hard at work.
```

File: _drafts/shortcut-long-pain.markdown [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+> * name (required): Emme
+> * color: brown
+> * size: medium
+> * species (required): dog
+>
+> Create an interface allowing a pet to be edited. The editable fields are
+> displayed in the order: name, species, color, and size. The order is
+> required to support [BEAUTIFUL INTERFACE].
+
+To reduce duplication you decide to iterate over collections of fields.
```
Comment (tdouce) [inline _drafts/shortcut-long-pain.markdown]:
```
To reduce duplication, you decide to iterate over the collections of fields.
```

Note: addition of `,` and `the`

File: _drafts/shortcut-long-pain.markdown [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  <%= f.text_field field, required: true %>
+<% end %>
+
+<%= f.text_field "color" %>
+<%= f.text_field "size", required: true %>
+<%= f.text_field "owner" %>
+~~~
+
+You have had to give up the convenience of looping over fields to display
+the last set. Further, it seems a little odd that "size", a required field, is
```
Comment (tdouce) [inline _drafts/shortcut-long-pain.markdown]: Perhaps:

```
You had to give up the convenience of looping over the fields to display the last set
```

Note: removed `have` and added `the`

**Pull request merged**